src/DatePicker.js

```diff
@@ -29,13 +29,17 @@ import cx from 'classnames';
   floating: t.maybe(t.Boolean),
   closeOnClickOutside: t.maybe(t.Boolean), // used only with DatePickerInput
   className: t.maybe(t.String),
+  prevIconClassName: t.String,
```

Comment:
@sawsanias here it was actually correct to leave the `maybe` as they are optional and when missing they're replaced with the values in `defaultProps`.

I don't think I commented these ones too, but if I did sorry :)